consoleme/models.py

```diff
@@ -489,9 +488,15 @@ class PolicyRequestModificationRequestModel(BaseModel):
     ]
 
 
+class ActionResult1(BaseModel):
```

Comment:
Looks like there are things in `consoleme/lib/v2/requests.py` that referenced `ActionResult`. I think those references need to be changed to `ActionResult1`

Reply:
All of those are okay to reference `ActionResult` since they don't have the `visible` field. I could potentially combine `ActionResult` and `ActionResult1` since `visible` is an optional bool on the latter. Thoughts?